packages/react-native-win32/etc/react-native-win32.api.md

```diff
@@ -435,6 +447,11 @@ export type SharedAccessibilityPropsIOSandWin32 = {
     onAccessibilityTap?: () => void;
 };
 
+// @public (undocumented)
+export type SharedTextPropsAndroidandWin32 = {
```

Comment:
>a [](start = 34, length = 1)

Just checking if it is intentional to use 'and' and not 'And' to match more common PascalCasing pattern.
I fully realize that this matches `IOSandWin32` above and am questioning that one too :)

Reply:
I don't know if this is intentional. I noticed this too and decided to match `IOSandWin32` for consistency but it does go against the pattern. I can update both if you feel that's best.